.woodpecker.star

```diff
@@ -266,6 +266,23 @@ def beforePipelines(ctx):
            pipelinesDependsOn(pnpmlint(ctx, "format"), pnpmCache(ctx))
 
 def stagePipelines(ctx):
+    if ctx.build.event == "pull_request" and "🎉 release" in ctx.build.title.lower():
+        return [{
```

Comment:
if this doens't work then let's keep it as before. And from there we can also skip the pipelines (ocis-cache, web-cache,...) related to unit and e2e tests.
```suggestion
    if ctx.build.event == "pull_request" and "🎉 release" in ctx.build.title.lower():
        return []
```